packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -299,12 +310,19 @@ class EditorPage {
 
 	// Attempts to find the given block button in the block inserter control.
 	async findBlockButton( blockName ) {
+		// Wait for the first block, Paragraph block, to load before looking for other blocks
+		const paragraphBlockLocator = isAndroid()
+			? '//android.widget.Button[@content-desc="Paragraph block"]/android.widget.TextView'
+			: '//XCUIElementTypeButton[@name="Paragraph block"]';
```

Comment:
For now, I've set that it waits for the Paragraph block to load because it's the first one on the blocks list, I'm not sure how often the list changes so this has the potential to fail in the future. Another option is to wait for the search box. What do you all think?

Reply:
The block picker view has a `testID` attribute with value `add-block-button`, maybe we should rely on that, wdyt?
As an alternative, we might consider adding accessibility info, which apart from helping us query the element, would be a good improvement in terms of a11y.

https://github.com/WordPress/gutenberg/blob/41325b983feabcc46615cd6b1a8a5efe64c5a9f0/packages/block-editor/src/components/inserter/index.native.js#L50

Reply:
Ah ok but looks like that is before the block picker view:
<img width="1277" alt="Screenshot 2022-03-28 at 11 01 44 AM" src="https://user-images.githubusercontent.com/17252150/160321542-adb9d078-6a2f-4aae-b312-9526ea93d91a.png">

After some investigation, looks like we can actually create another wait helper that uses IDs! But it has to be an ID defined like this (i.e For Android, `android:id/{value}`): 
<img width="1277" alt="Screenshot 2022-03-28 at 11 42 16 AM" src="https://user-images.githubusercontent.com/17252150/160322987-75941cd6-e7fd-45ce-9274-2f7af06c3cd9.png">

There is an open issue for this [here](https://github.com/facebook/react-native/issues/32237). Appium doc how to use this [here](https://appium.io/docs/en/commands/element/find-elements/). 

I am not sure where this is defined (looks like not on the `./index.native.js` file), if we can add an ID on a more appropriate view we could depend on that to load first before continuing the test.

I think, for now, I will leave it to wait for Paragraph block before continuing the test. Future improvements (i.e. add id and use it for the test) to be done in a separate PR. Is that ok?

Reply:
Oh sorry, I'm afraid I referenced the wrong test ID. The one I'm wanted to share is `InserterUI-Blocks`:

https://github.com/WordPress/gutenberg/blob/41325b983feabcc46615cd6b1a8a5efe64c5a9f0/packages/block-editor/src/components/block-types-list/index.native.js#L89

In any case, I think it's quite reliable to use the Paragraph block as it's a block that will be always present.